src/components/UserProfile/UserProfileModal/__tests__/EditConfirmModal.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+import { userProfileMock } from '../../../../__tests__/mockStates';
+import EditConfirmModal from '../EditConfirmModal';
+
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
```

Comment:
Please consider removing this function, it appears to be unused.

Reply:
Thak you for pointing it out.
Was added to test a different feature.
Will remove it.